frontend/src/api/assignments.ts

```diff
@@ -18,28 +18,34 @@ export const getAssignments = async (
     throw new Error("Network response was not ok");
   }
 
-  const data = (await result.json())
+  const data = await result.json();
 
-  return data.assignment_outlines as IAssignmentOutline[]
+  return data.assignment_outlines as IAssignmentOutline[];
 };
 
 export const getAssignmentIndirectNav = async (
-  classroomid: number, assignmentID: number
+  classroomid: number,
+  assignmentID: number
 ): Promise<IAssignmentOutline> => {
   const base_url: string = import.meta.env.VITE_PUBLIC_API_DOMAIN as string;
-  const result = await fetch(`${base_url}/classrooms/classroom/${classroomid}/assignments/assignment/${assignmentID}`, {
-    method: 'GET',
-    credentials: 'include',
-    headers: {
-      'Content-Type': 'application/json',
-    },
-  });
+  const result = await fetch(
+    `${base_url}/classrooms/classroom/${classroomid}/assignments/assignment/${assignmentID}`,
+    {
+      method: "GET",
+      credentials: "include",
+      headers: {
+        "Content-Type": "application/json",
+      },
+    }
+  );
 
   if (!result.ok) {
-    throw new Error('Network response was not ok');
+    throw new Error("Network response was not ok");
```

Comment:
Probably better to return direct error unless this is an example of masking "secure" stack traces

Reply:
'm not sure what use there would be to potentially exposing the stack trace and error information if the error isn't caught.